examples/bert_ptq_cpu/mlflow_bert_config.json

```diff
@@ -0,0 +1,137 @@
+                "aml_docker_config": {
+                    "base_image": "mcr.microsoft.com/azureml/openmpi4.1.0-ubuntu20.04",
+                    "conda_file_path": "conda.yaml"
+                },
+                "is_dev": true
+            }
+        },
+        "docker_system": {
+            "type": "Docker",
+            "config": {
+                "local_docker_config": {
+                    "image_name": "olive-image",
+                    "build_context_path": "docker",
+                    "dockerfile": "Dockerfile"
+                },
+                "is_dev": true
+            }
+        }
+    },
+    "evaluators": {
+        "common_evaluator": {
+            "metrics":[
+                {
+                    "name": "accuracy",
+                    "type": "accuracy",
+                    "sub_types": [
+                        {"name": "accuracy_score", "priority": 1, "goal": {"type": "max-degradation", "value": 0.01}},
+                        {"name": "f1_score"},
+                        {"name": "auc", "metric_config": {"reorder": true}}
+                    ]
+                },
+                {
+                    "name": "latency",
+                    "type": "latency",
+                    "sub_types": [
+                        {"name": "avg", "priority": 2, "goal": {"type": "percent-min-improvement", "value": 20}},
+                        {"name": "max"},
+                        {"name": "min"}
+                    ]
+                }
+            ]
+        }
+    },
+    "passes": {
+        "conversion": {
+            "type": "OnnxConversion",
+            "config": {
+                "target_opset": 13
+            }
+        },
+        "transformers_optimization": {
+            "type": "OrtTransformersOptimization",
+            "config": {
+                "model_type": "bert",
+                "num_heads": 12,
+                "hidden_size": 768,
+                "float16": false
+            }
+        },
+        "perf_tuning": {
+            "type": "OrtPerfTuning",
+            "config": {
+                "input_names": ["input_ids", "attention_mask", "token_type_ids"],
+                "input_shapes": [[1, 128], [1, 128], [1, 128]],
+                "input_types": ["int64", "int64", "int64"]
+            }
+        }
+    },
+    "engine": {
+        "log_severity_level": 0,
+        "search_strategy": {
+            "execution_order": "joint",
+            "search_algorithm": "tpe",
+            "search_algorithm_config": {
+                "num_samples": 3,
+                "seed": 0
+            }
+        },
+        "evaluator": "common_evaluator",
+        "host": "local_system",
+        "target": "local_system",
+        "execution_providers": ["CPUExecutionProvider", "OpenVINOExecutionProvider"],
+        "clean_cache": true,
+        "cache_dir": "cache",
+        "packaging_config": {
+            "extra_config": {
+                "export_model_in_mlflow_format": true
```

Comment:
Could we leverage the model type designed for input model here?

Reply:
Since the only output format supported is onnx.mlflow, it's better not to intruduce the model file format here.